package.json

```diff
@@ -2,35 +2,30 @@
   "name": "@jupiterone/integration-sdk",
   "version": "0.15.0",
   "description": "The SDK for developing JupiterOne integrations",
-  "main": "dist/framework/index.js",
-  "types": "dist/framework/index.d.ts",
+  "main": "src/framework/index.js",
+  "types": "src/framework/index.d.ts",
   "repository": "git@github.com:JupiterOne/integration-sdk.git",
   "author": "JupiterOne <dev@jupiterone.io>",
   "license": "MPL-2.0",
   "engines": {
-    "node": "10.x || 12.x"
+    "node": "10.x || 12.x || 14.x"
   },
   "publishConfig": {
     "access": "public"
   },
   "bin": {
     "j1-integration": "./bin/j1-integration"
   },
-  "files": [
-    "dist",
-    "bin",
-    "testing.js",
-    "testing.d.ts"
-  ],
   "scripts": {
     "lint": "eslint . --cache --fix --ext .ts,.tsx",
     "format": "prettier --write '**/*.{ts,js,json,css,md,yml}'",
     "type-check": "tsc --noEmit",
     "test": "jest",
     "test:ci": "yarn lint && yarn type-check && yarn test --ci",
-    "prebuild": "yarn test:ci",
-    "build": "tsc -p tsconfig.dist.json --declaration",
-    "prepublishOnly": "yarn build"
+    "compile": "tsc -p tsconfig.dist.json --declaration",
+    "prebuild": "rm -rf dist",
+    "build": "yarn compile && cp -R bin README.md LICENSE yarn.lock package.json ./dist",
```

Comment:
I rather miss the notion of using `files` to specify what goes into the built package. Seems so clean and native.

I think `yarn.lock` is not necessary or useful in the npm package, is it?

Reply:
Well... I use it to do a production install when linking so it helps make sure the versions that were specified work properly. It's probably not strictly necessary tho.

Reply:
Regarding `files`, I didn't think it was necessary because we're already packaging exactly what we need into the `dist` folder. I feel like the `files` entry will just add confusion if something is not packaged up as expected because they forgot to add entry to `files`.

Reply:
Yup, it's undesirable to specify `files` in this case for sure. I just don't fully understand why that mechanism is not used instead of this approach, which I assume has to do with `autobuild`, which I have never used.